components/n-ui/tracking/ft/events/audio-teaser-view.js

```diff
@@ -0,0 +1,22 @@
+const oTracking = require('o-tracking');
+
+const audioTeaserView = {
+	init: () => {
+		const getContentIdFromXTeaser = el => el.getAttribute('data-id');
+		const getContentIdFromNTeaser = el => el.querySelector('.o-teaser__heading a').getAttribute('data-content-id');
```

Comment:
~As this code lives at a global level, it might be wise add a check that `el.querySelector('.o-teaser__heading a')` exists before running `getAttribute()` on it.~

On second thoughts I think we want this to throw so we know about it going wrong?

Reply:
There is not such a deep thought... 🙈
However, I agree with you. We should keep as it is.  `o-teaser--audio` should have `.o-teaser__heading a`. If it didn't had the element, it meant something went really wrong.